modules/meeting/app/components/meetings/side_panel/details_component.html.erb

```diff
@@ -50,7 +50,7 @@
           details.with_row do
             render_meeting_attribute_row(:calendar) do
               render(Primer::Beta::Text.new) do
-                format_date(@meeting.start_time)
+                format_time_as_date(@meeting.start_time)
```

Comment:
Nothing in method names differentiate `format_date` from `format_time_as_date` to indicate that one purely formats, and the other one uses current user's time zone.
Same for `format_time`: it uses user's time zone if possible, but that's impossible to guess from the name

Do you mind if we create a maintenance ticket to rename `format_time_as_date` as `format_time_as_user_date` (or even `format_user_date`), and `format_time` as `format_user_time`?

Reply:
we could also change the method signature, and if you pass a date, we automatically assume you want it in the user's time zone?

Reply:
Discussed and decided: keep `format_date` and `format_time`, allow both to receive `time_zone: override`, always default to user's time zone.

tracked in https://community.openproject.org/projects/openproject/work_packages/63911